### DISCUSSION TITLE
How to remove "Users must not work more than 9 hours without a 45 minute break"

### DISCUSSION THREAD

**Discussion by Sarmingsteiner** [KatjaGlassConsulting/ApprovalBundle#51] [General]
Since the rule "Users must not work more than 9 hours without a 45-minute break" is generally not applicable in Austria, could you please point us to the relevant section in the code where we can remove it ourselves?

**Comment by KatjaGlassConsulting** [comment]
Dear @Sarmingsteiner , this can be done in the settings tab:

![image](https://github.com/user-attachments/assets/f9f528f6-a69e-4c5c-8aba-31f060215e27)

![image](https://github.com/user-attachments/assets/e1fdb09c-fb82-40e1-9f72-2d4cd250e50f)

**Comment by KatjaGlassConsulting** [comment]
If you want to keep the other time checks, you can update the source at:
Toolbox/BreakTimeCheckToolGER.php

Comment out the specific check call:
`$this->checkNineHoursWithoutBreak($timesheets, $errors);` to `//$this->checkNineHoursWithoutBreak($timesheets, $errors);`
